plone/app/discussion/browser/javascripts/moderation.js

```diff
@@ -24,10 +24,10 @@ require([  // jshint ignore:line
     //#JSCOVERAGE_IF 0
 
     /**************************************************************************
-     * Window Load Function: Executes when complete page is fully loaded,
+     * Document Ready Function: Executes when complete page is fully loaded,
      * including all frames,
      **************************************************************************/
-    $(window).load(function () {
+    $(document).ready(function () {
```

Comment:
Isn't `$(document).ready` fired before `$(window).load`? If this is what you want I am fine with that. Comment might be updated.

Reply:
I'm agree with you $(document).ready is the first, but $(window).load it doesn't always work especially on FF and IE. Instead $(document).ready it seems to work properly. 

But maybe it’s better move this modify to another PR.

Reply:
As you prefer :) I'm not so much into javascript!

Reply:
done :)